boards/arm/degu_evk/Kconfig.defconfig

```diff
@@ -0,0 +1,103 @@
+# Kconfig - Degu Evaluation Kit configuration
+#
+# Copyright (c) 2019 Atmark Techno, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_DEGU_EVK
+
+config BOARD
+	default "degu_evk"
+
+if USB
+
+config USB_NRF52840
+	def_bool y
+
+config USB_DEVICE_STACK
+	def_bool y
+
+endif # USB
+
+if IEEE802154
+
+config IEEE802154_NRF5
+	def_bool y
+
+endif # IEEE802154
+
+config I2C
+	def_bool y
+
+if I2C
+
+config I2C_NRFX
+	def_bool y
```

Comment:
Similar story as with `ADC_NRFX_SAADC`. You don't need to do it, the option is already set to `y` by default, see [drivers/i2c/Kconfig.nrfx](https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/i2c/Kconfig.nrfx#L10).